### ISSUE TITLE
Scan config ignores deselection of SSL/TLS item

### ISSUE THREAD

**Issue by kcadmin-github** [immauss/openvas#147]
I want to create a new scan config which is a complete clone of "full and fast" but with the SSL/TLS item "report vulnerable cipher suites for https" disabled.

- Created a clone of "full and fast"
- Deselected the appropriate item
- It just reappears after some seconds

![003_20220913_100408_deselect](https://user-images.githubusercontent.com/69455268/190369529-268d3608-abd2-42aa-b3fa-835748e9e90e.png)
![004_20220913_100412_deselected](https://user-images.githubusercontent.com/69455268/190369534-09894f1b-2c71-4270-b215-6d3ed33ddecb.png)
![005_20220913_100419_reapperance](https://user-images.githubusercontent.com/69455268/190369541-f828f195-274f-468b-a2e6-7a0c26080140.png)

**Comment by immauss** [comment]
I'm seeing the same behavior. I suspect that some other test that is still enabled is dependent upon that one. This is more a question to be asked over in the community forums though as it's more GVMd/Greenbone related that container related. 

https://forum.greenbone.net/

I'm curious though. Please let me know if you find anything. 

Thanks,
Scott

**Comment by kcadmin-github** [comment]
In the meantime I did a workaround.

- I deselected all of the SSL/TLS group and saved it. That was accepted.
- Then I enabled item by item until 6 less than the full 78 items (I found also 5 additional items I'm not interested in).

For me this approach was enough.

Unfortunately it's not telling any dependencies when deselecting this item, that's maybe something Greenbone needs to add/fix.

Thanks anyway for the comment.
